services/beeper-connector/UPGRADE.md

```
@@ -0,0 +1,162 @@
+
+```bash
+npm run sync-to-evault    # Sync Beeper → eVault
+npm run sync-from-evault  # Sync eVault → Beeper
+npm run realtime          # Real-time bidirectional sync
+npm run export-rdf        # TypeScript RDF export
+```
+
+## Architecture Changes
+
+### v1.0 (Python)
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)